### ISSUE TITLE
Connection to new site refused

### ISSUE THREAD

**Issue by jobberz77** [nvisionative/nvQuickSite#176]
Hello,

When I create a site following the steps as described on the wiki page and click the visit site button it refuses to connect to the site. I have my SQL Server management running and the database is being displayed in the solution explorer. 

This is most likely a mistake on my end, but I am not sure what went wrong here.

![image](https://user-images.githubusercontent.com/23524035/45972151-4818fa00-c03b-11e8-9d3f-96fb90a6b408.png)

**Comment by nvisionative** [comment]
@jobberz77 our guess at this point is that your HOSTS file was not accessible for nvQuickSite to modify it.  Take a look at #171 for more details on what the causes may be, but in short, others have seen issues with anti-virus software preventing modification to the HOSTS file.

**Comment by nvisionative** [comment]
This could also be an IIS related issue.  Please check out [this nice article by Clint Patterson on IIS configuration](http://www.dnnsoftware.com/community-blog/cid/155489/iis-gotchas-when-installing-dnn).

**Comment by jobberz77** [comment]
Thanks for your reply! I followed the IIS configuration article and some of those features were turned off. However, when I tried to create a new site after fixing the configuration, it still was not working for me.

After this, I tried creating a site in  "C:/Websites/....", and I changed the url to end with ".loc" instead of " .dev", this fixed it for me!

After some more testing, I figured out that for some reason, URLs that end with ".dev" will refuse communication, while URLs that end with ".loc" work fine. The location where the site is created did not matter(except for the CSS not showing properly when using a directory located on the Desktop).
